src/jit/codegencommon.cpp

```diff
@@ -5228,6 +5228,13 @@ void CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg, bool* pXtraRegClobbere
 
                 emitAttr size = emitActualTypeSize(destMemType);
 
+#if defined(_TARGET_ARM64_) && FEATURE_SIMD
```

Comment:
This needs to be:
```
#if defined(_TARGET_ARM64_) && defined(FEATURE_SIMD)
```
(it's annoying, and confusing, that some are `#if` and some are `#ifdef`...)

Reply:
Fixed NIT